core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -880,6 +880,28 @@ export class BaseUser {
     this.page = newPage;
     return newPage;
   }
+
+  /**
+   * Scrolls to the bottom of the page.
+   */
+  async scrollToBottomOfPage(): Promise<void> {
+    await this.page.evaluate(async () => {
+      await new Promise<void>(resolve => {
+        let totalHeight = 0;
+        const distance = 100;
```

Comment:
Always state the units for dimensional quantities. In this case, distanceInPixels

Reply:
Check comment below.